css/amp-story-player-iframe.css

```diff
@@ -59,3 +59,19 @@ main .story-player-iframe[i-amphtml-iframe-position="1"] {
 main .story-player-iframe[i-amphtml-iframe-position="-1"] {
   transform: translate3d(-100%, 0, 0);
 }
+
+.amp-story-player-close-button {
```

Comment:
/cc @newmuis do you think these should be private classes? (i.e. prefixed with `.i-amphtml-`)?

Reply:
This prefix is helpful with the AMP validator, but since this code doesn't get validated, it would just be a convention at most.

Also, I think it's also quite possible that we'd want to allow publishers to style this?